openfisca_france/model/revenus/remplacement/indemnites_journalieres_securite_sociale.py

```diff
@@ -74,3 +73,9 @@ def function(self, simulation, period):
         )
 
         return period, result
+
+class date_arret_de_travail(Variable):
+    column = DateCol(default = nan)
```

Comment:
You should put a default date that do not modify the results previous the introduction of the feature. Here I would use the equivalent of + infinity for dates.
@fpagnoux do you agree ?